Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.m

```diff
@@ -264,31 +264,22 @@ - (BOOL)tableView:(UITableView *)tableView canMoveRowAtIndexPath:(NSIndexPath *)
 
 #pragma mark - Fetched results controller
 
-- (NSFetchedResultsController *)fetchedResultsController
-{
-    if (!_fetchedResultsController) {
-        [self initFetchedResultsController];
```

Comment:
Without this lazy-loader, is `initFetchedResultsController` being called somewhere else?